packages/lib-panoptes-js/vitest.config.js

```diff
@@ -0,0 +1,8 @@
+import { defineConfig } from 'vitest/config'
+
+export default defineConfig({
+  test: {
+    globals: true,
+    environment: 'jsdom',
+    setupFiles: './test/setup.js'  }
+})
```

Comment:
Note the difference in this file compared to Vitest setup in packages that use React. There's no need for `@vitejs/plugin-react` in lib-panoptes-js.